### DISCUSSION TITLE
issue with @types[jest-when] and mockResolvedValue

### DISCUSSION THREAD

**Discussion by doombski** [DefinitelyTyped/DefinitelyTyped#67453] [Issues with a @types package]
hello, I am experiencing an issue with jest-when concerning mockResolvedValue and utilizing it with calledWith.  My promises all timeout without resolution. Current quick fix has been to use .mockReturnValue and wrap the expected output in a Promise.resolve

[a comment by an automated account is not shown]

**Comment by boginw** [comment]
Hi @bradomski, I'm sorry to hear you're experiencing issues with jest-when. However, this repository only describes the types used in jest-when and doesn't have control over how jest-when functions. You'll have to file your issue on jest-when's Github repository: https://github.com/timkindberg/jest-when
